src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int maxNumber = 100;
```

Comment:
Don’t forget how to name constants according to style guide (the same with naming of enum values)
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names